components/UI/modal/modal.module.scss

```diff
@@ -0,0 +1,28 @@
+  
+.Modal {
+  position: fixed;
+  z-index: 500;
+  background-color: white;
```

Comment:
will background-color not be white by default  :thinking:

Reply:
What if we add fractional opacity here along with white color if we're going with it? or blue the background maybe? For blurring, I found this https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur()
but isn't supported by IE

Reply:
This is needed. Understand what it's trying to do here.
Whenever we open the Modal, we show a backdrop that has a background black with a slight opacity.
Now, the modal is an overlay on top of that, so if we do not have to apply a background of white, this is how the Modal will look

<img width="1422" alt="Screenshot 2021-09-02 at 3 19 28 PM" src="https://user-images.githubusercontent.com/29303618/131822765-1821ecd3-b20e-4025-9de8-a81267045c77.png">

And if we have a background of white, this is how it will look

<img width="1426" alt="Screenshot 2021-09-02 at 3 20 11 PM" src="https://user-images.githubusercontent.com/29303618/131822867-4a5ba27c-ba69-4b13-b50d-fa65800d2de8.png">

Reply:
Got it thank you for explaining 😃